src/main/java/org/sonarlint/intellij/core/ProjectBindingManager.java

```diff
@@ -81,15 +83,15 @@ public synchronized ConnectedSonarLintEngine getConnectedEngine() {
     return engineManager.getConnectedEngine(notifications, serverId, projectKey);
   }
 
-  public synchronized SonarQubeServer getSonarQubeServer() {
+  public synchronized SonarQubeServer getSonarQubeServer() throws InvalidBindingException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.sonarlint.intellij.exception.InvalidBindingException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)